recipes/napari-deepmeta/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "napari-deepmeta" %}
+{% set version = "1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-deepmeta-{{ version }}.tar.gz
+  sha256: dab7a521af490d865ee8aff82f41e7ec4016d43b8b3e65247cad65e401f7f0e7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - setuptools-scm
+  run:
+    - connected-components-3d
+    - napari ==0.4.8
```

Can we relax this a little? :) 

```suggestion
    - napari >=0.4.8
```

---

@EdgarLefevre

---

Hmm not sure, maybe the plugin has not been tested in a while.